### PR TITLE
add joined field + lat/lon column

### PR DESCRIPTION
**Pull request by SebastienPeillet**

### CODE REVIEW

File: src/gui/export.py [hunk 1]
```diff
@@ -6,15 +6,20 @@
...
     QToolBar,
     QFileDialog,
 )
 
-from qgis.core import QgsVectorFileWriter
+from qgis.core import (
+    QgsVectorFileWriter,
+    QgsVectorLayer,
+    QgsField,
+    QgsVectorLayerJoinInfo,
```
Comment (pblottiere) [inline src/gui/export.py:21]: import ordre

File: src/gui/export.py [hunk 2]
```diff
@@ -59,14 +64,71 @@ def export(self):
             self.progressBar.setFormat(
                 f"Export de la couche {layer.name()}, Total : %p%"
             )
+
+            # Export is done from copy to avoid bug with join field, due to
+            # the table dock.
+            layer = QgsVectorLayer(layer.source(), layer.name())
+
+            # Add Lon/Lat field
+            if layer.geometryType() != 4:
```
Comment (pblottiere) [inline src/gui/export.py:73]: Can we use an enum instead of the `4` value?

Comment (SebastienPeillet) [reply src/gui/export.py]: :+1:

**Comment by SebastienPeillet** [comment]
@pblottiere it's ready to be merged ! :)

**Comment by pblottiere** [comment]
Good thanks :+1:

**Pull request merged**